community/farbfeld/APKBUILD

```diff
@@ -7,7 +7,7 @@ pkgdesc="Lossless image format which is easy to parse, pipe and compress"
 url="https://tools.suckless.org/farbfeld/"
 arch="all"
 license="ISC"
-depends=""
+depends="file"
```

Comment:
Just a note in case you want to submit more patches in the future: You need to bump the value of the `pkgrel` variable, otherwise changes like this one won't have an effect. I fixed this for you when merging this change.